Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -2659,18 +2659,18 @@ static WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
 
 std::optional<Ref<WebExtensionSidebar>> WebExtensionContext::getSidebar(WebExtensionWindow const& window)
 {
-    if (auto *windowAction = m_sidebarWindowMap.get(window))
-        return *windowAction;
+    if (auto *windowSidebar = m_sidebarWindowMap.get(window))
```

Comment:
i think it's preferred to `RefPtr` instead of `auto *` here. we had feedback previously to not use `auto` for `Ref` and `RefPtr` types

Reply:
In this case, although `m_sidebarWindowMap` has value type `Ref<WebExtensionSidebar>`, it seems as though `m_sidebarWindowMap.get` is returning a raw `WebExtensionSidebar *`. Would it still be advantageous to specify the full type here?

Reply:
hmm i'm not sure why that is. cc @xeenon

Reply:
It would be best to do:
```c++
if (RefPtr windowSidebar = m_sidebarWindowMap.get(window))
```
So the local holds a reference and it can't disappear from under you. This is an area we don't do well with everywhere yet.

Reply:
Odd, I swear that was generating a compiler error for me about 30 minutes ago, but it seems to have resolved itself / I was doing something wrong before. Adding this change now.